python/tvm/topi/cuda/dense.py

```diff
@@ -77,8 +77,8 @@ def _callback(op):
 
 
 def _schedule_dense_small_batch(cfg, s, C):
-    A, _ = C.op.input_tensors
-    _, in_dim = get_const_tuple(A.shape)
+    A, weights = C.op.input_tensors
```

Comment:
```suggestion
    _, weights = C.op.input_tensors
```

`A` is no longer needed here.